### DISCUSSION TITLE
Redux toolkit query

### DISCUSSION THREAD

**Discussion by jacksonjp** [monstar-lab-oss/admin-panel-template-reactjs#102] [Ideas]
https://redux-toolkit.js.org/rtk-query/overview

**Comment by yukinoda** [comment]
[Example - Migrating data-fetching logic from Redux Toolkit to RTK Query](https://redux-toolkit.js.org/rtk-query/usage/migrating-to-rtk-query#example---migrating-data-fetching-logic-from-redux-toolkit-to-rtk-query)
